Automattic/jetpack
# Site-Level User Profile: Show links back to WordPress.com on profile.php

Fixes https://github.com/Automattic/dotcom-forge/issues/8421

## Proposed changes:


This PR shows links back to WordPress.com for them to manage their WordPress.com profile. 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Simple Classic & Default

* If you test Simple **Default** sites, set `wpcom_site_level_user_profile` option 1
* Go to `/wp-admin/profile.php`
* Observe "Contact Info > Email" is disabled and has a link to /me/account
	<img width="1105" alt="Screenshot 2024-07-31 at 15 27 03" src="https://github.com/user-attachments/assets/ddea7300-6005-4408-a566-c3f2f99fb36f">
* Observe "Account Management > New Password" has a link to /me/security
	<img width="778" alt="Screenshot 2024-07-31 at 15 32 14" src="https://github.com/user-attachments/assets/0fae1003-7976-4d05-963f-1dd94183e979">


Atomic Classic

* Go to `/wp-admin/profile.php`
* Observe "Contact Info > Email" has a link to /me/account
	<img width="1100" alt="Screenshot 2024-07-31 at 15 25 50" src="https://github.com/user-attachments/assets/aa5760ff-095a-46bf-a2a0-e5e41e9ecab0">
* Observe "Account Management > New Password" has a link to /me/security
	<img width="782" alt="Screenshot 2024-07-31 at 15 26 01" src="https://github.com/user-attachments/assets/78ea2013-6ee7-461a-9c6c-2257cb7ba507">

- [ ] For Atomic Default, we need to address this issue first: https://github.com/Automattic/dotcom-forge/issues/8511
